src/Seq.App.HttpRequest/HttpApp.cs

```diff
@@ -28,15 +28,20 @@ internal HttpApp(HttpAppClient client)
         }
         
         [SeqAppSetting(DisplayName = "URL",
+            Syntax = "template",
```

Comment:
I don't think we have provided enough documentation for people to be able to use this. From the HelpText the user knows that they can use event properties, but what about the functions that are in scope, e.g. if they want to url encode that email, or use some time functions etc. There are the [alert properties](https://docs.datalust.co/docs/alert-properties), but it is hard to find, only relates to alerts, and doesn't include everything.

Reply:
Yep, agreed about that. I think we should consider this app as a bit of a trial/forcing function to fuel the process of iteration - from here forwards, each release of this app, and of Seq itself, can push usability forwards a bit.

This approach also opens up the possibility that Seq itself could provide (or link to) detailed help for `template` and `expression` settings 🤔